test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/V3DetailControlModelTests.cs

```diff
@@ -131,6 +131,20 @@ public void VulnerabilityCountWhenMetadataHasVulnerability_Calculated()
             Assert.Equal(_testInstance.PackageVulnerabilityCount, _testData.TestData.Vulnerabilities.Count());
         }
 
+        [Fact]
+        public void PackageVulnerabilitiesWhenMetadataHasVulnerability_OrderedBySeverityDescending()
```

Comment:
Please rename according to https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming

Reply:
Thanks for pointing that out! I was following convention in the other tests in this class, which are wrong. 😄 Fixed it now to comply with our guidelines. Let me know if I missed something.